fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/service/ExternalServicesPropertiesReadPlatformServiceImpl.java

```diff
@@ -87,8 +87,6 @@ public SMTPCredentialsData extractData(final ResultSet rs) throws SQLException,
                     password = value;
                 } else if (ExternalServicesConstants.SMTP_HOST.equalsIgnoreCase(name)) {
                     host = value;
-                } else if (ExternalServicesConstants.SMTP_PORT.equalsIgnoreCase(name)) {
-                    port = value;
                 } else if (ExternalServicesConstants.SMTP_USE_TLS.equalsIgnoreCase(name)) {
```

Comment:
Why are we no longer reading the port from the database?

Reply:
To send gmails it requires either port 587 or 465. The port stored in database is 25.

Reply:
Hmm.. why would we not simply store the right port number in the database when using Gmail?

Reply:
Yeah, we could change that. But I need to verify whether these changes are good to proceed.

Reply:
@BLasan,, perhaps you want to set the port in the database as @ptuomola suggested.